doc/source/preparing_releases_and_hotfixes.rst

```diff
@@ -0,0 +1,206 @@
+.. -*- coding: utf-8 -*-
```

Comment:
Speaking as the release manager, I don't think we do hotfixes?  I think I'd sooner just push out a new release and call the old one broken.

Reply:
Fair enough --[ removed that section](https://lilyminium.github.io/UserGuide/preparing_releases_and_hotfixes.html).